guides/models.md

```diff
@@ -77,4 +80,20 @@ class Customer < ActiveRecord::Base
     # some calculation
   end
 end
+
+# app/models/concerns/commentable
+module Commentable
+  # Concern should be named with `able` prefix
```

Comment:
I'd suggest moving this message to the line about the concerns above (in Key Points section) or add a `Conventions` (like in some other articles - https://github.com/ergoserv/handbook/blob/master/guides/query_objects.md#conventions).